opm/flowdiagnostics/TracerTofSolver.cpp

```diff
@@ -271,15 +271,22 @@ namespace FlowDiagnostics
 
     void TracerTofSolver::solveSingleCell(const int cell)
     {
-        // Compute influx.
+        // Compute influx (divisor of tof expression).
         double source = 2.0 * source_term_[cell];  // Initial tof for well cell equal to half fill time.
         if (source == 0.0 && is_start_[cell]) {
             source = std::numeric_limits<double>::infinity(); // Gives 0 tof in start cell.
         }
-        const double total_influx_ = influx_[cell] + 2.0 * source_term_[cell];
+        const double total_influx_ = influx_[cell] + source;
```

Comment:
Completely unrelated to the semantics of this change: why does this local variable (`total_influx_`) have an underscore suffix?